akka-docs/src/main/paradox/distributed-data.md

```diff
@@ -212,6 +212,10 @@ from other nodes might not be visible yet.
 When using @scala[`WriteLocal`]@java[`writeLocal`] the update is only written to the local replica and then disseminated
 in the background with the gossip protocol, which can take few seconds to spread to all nodes.
 
+When using `ReadLocal`, you will never receive a `GetFailure` response, since the local replica is always available to
+local readers. `WriteLocal` however may still reply with `UpdateFailure` messages, in the event that the merge function
+threw an exception, or, if using durable storage, if storing failed.
+
```

Comment:
This is correct, good clarification.

`Get` can reply with `GetSuccess`, `NotFound` or `DataDeleted`.

`Update` can reply with `UpdateSuccess`, `ModifyFailure` or `DataDeleted`
`Update` can also reply with `StoreFailure` if durable.